.github/workflows/update-browserlist.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Browserlist DB
```

Comment:
Is this Github Action workflow essentially the same for all repos that use browserslist? If so, should we define the workflow once in a shared place that can be used by many repos?

Like what we do with the shared commit linting Github Action workflow file across all the repos: https://github.com/openedx/paragon/blob/master/.github/workflows/lockfileversion-check.yml#L11-L13

The shared Github Action workflows are defined here: https://github.com/openedx/.github/tree/master/workflow-templates, and should be accessible across repos without having to duplicate workflow files, resulting in more maintainable code.

Also, nit: let's change "browserlist" -> "browser**s**list" to match the actual name of browserslist.

Reply:
yeah it seems a right way to do this, let me create a PR there

Reply:
created [this](https://github.com/openedx/.github/pull/29) PR for shared workflow

Reply:
@abdullahwaheed Left some comments on the PR for the shared workflow!

Reply:
yes, updating that PR on the basis of your review